lib/annotated/init.go

```diff
@@ -431,3 +431,18 @@ func (cs *acs) deploy(ctx context.Context) (err error) {
 
 	return nil
 }
+
+func (cs *acs) Reload() error {
```

Comment:
what calls this?

Reply:
i think nothing in here directly, but somewhere there's something where if it keeps getting errors on `head` calls, it'll eventually try a reload to see if e.g. the database has been restarted and the conn is hanging or similar